### DISCUSSION TITLE
How to use my own orthophoto instead of using TMS for segmentation？

### DISCUSSION THREAD

**Discussion by yao-yu1999** [opengeos/segment-geospatial#99] [Q&A] [answered]
Hi! I took the aerial imagery myself and exported the orthophoto via software. I want to use my own orthophotos for segmentation instead of loading a map to download tiles.

**Comment by giswqs** [comment, marked as answer]
See this example. 
https://samgeo.gishub.org/examples/automatic_mask_generator/
```python
image = '/path/to/your/own/image.tif'
```
